package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mux.js",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
Rebase on the latest development branch. This change and the merge commits  should not be included in your PR.